tests/unit-tests/mesh/dim2/adjacency_graph/adjacency_graph.cpp

```diff
@@ -54,7 +54,7 @@ void add_unidirectional_edge(
 TEST(AdjacencyGraphTest, DefaultConstructor) {
   specfem::mesh::adjacency_graph<specfem::dimension::type::dim2> graph;
 
-  EXPECT_TRUE(graph.empty());
```

Comment:
This entire test doesnt do anything? 
```suggestion
  EXPECT_TRUE(boost.num_vertices(graph.graph()) == 0);
```